src/pages/projectSettings/tabs/NotificationsTab/transformers.test.ts

```diff
@@ -1,111 +1,215 @@
-import { ProjectSettingsInput, RepoSettingsInput } from "gql/generated/types";
+import { ProjectSettingsInput } from "gql/generated/types";
```

Comment:
Random comment placement: 
It looks like it can't handle the re-noify interval. 
![image](https://user-images.githubusercontent.com/13104717/178789255-11e2db66-c257-4dd2-98c0-f0dd0c5629be.png)
![image](https://user-images.githubusercontent.com/13104717/178789300-ffbfa5c3-2389-4116-a792-660896359b32.png)